packages/block-editor/src/store/test/reducer.js

```diff
@@ -3328,7 +3328,7 @@ describe( 'state', () => {
 			expect( state.clientIds ).toEqual( [ clientIdOne, clientIdTwo ] );
 		} );
 
-		it( 'should return client ids of all blocks when inner blocks are replaced with REPLACE_INNER_BLOCKS', () => {
+		it( 'should not return client ids of all blocks when inner blocks are replaced with REPLACE_INNER_BLOCKS', () => {
```

Comment:
Can we in any way make this a positive assertion?

Reply:
Yes. But some initial blocks in state. Freeze them. Run the reducer function. Check the blocks in state are the same.

Reply:
I've updated the test. Does this look better?

Reply:
Yes thank you so much 🙏🏻